README.md

```diff
@@ -44,6 +44,31 @@ Update the Trello secrets in the .env.template file and or the .env file.
 
 The lists used in Trello for testing were: "Not Started", "In Progress", "Peer Review", "On Hold", "Completed".
 
+## Testing with PyTest
+The lists used in Trello for testing were: "Not Started", "In Progress", "Peer Review", "On Hold", "Completed", "To Do", "Doing", "Done" - in that order left to right on trello.
+
+For Not Started, include 3 cards called "Item 1", "Item 9" and "Item 10" (in that order)
+
+For "In Progress" create 2 cards called "Item 5" and "Item 7" (In that order)
+
+For "Peer Review" create 2 cards called "Item 4" and "Item 6" (In that order)
+
+For "On Hold" create 1 card called "Item 2"
+
+For "Completed" create 1 card called "Item 3"
+
+for "To Do" create 1 card called "To Do Card"
+
+for "Doing" create 1 card called "Doing Card"
+
+For "Done" create 1 card called "Done Card"
```

Comment:
We shouldn't need to tell users how to set up their Trello boards before our tests run, otherwise we're very likely to have flaky tests (e.g. if a developer's adding a new test that moves cards, they're going to be constantly rearranging the board until they have everything right)
Instead we should either mock out Trello so that we're not really relying on it, or set up & tear down the board programmatically as part of our tests; the intention for our integration tests is to mock it out, while the end-to-end tests talk through setting up & deleting a board